docs/making-your-own.md

```diff
@@ -116,15 +103,16 @@ This part is important, users must have a method of verifying the image. The Lin
 
 ## Modification 
```

Comment:
This section is somewhat lackluster. There should be a general place for examples and instructions on modifying custom images, with the startingpoint README being preserved for startingpoint-specific instructions.